chapter-1-passing-tests.zig

```diff
@@ -640,12 +634,12 @@ fn dump(args: anytype) void {
 }
 
 test "tuple" {
-    const values = .{ 
+    const values = .{
         @as(u32, 1234),
         @as(f64, 12.34),
         true,
-        "hi"
-    } ++ .{ false } ** 2;
+        "hi",
+    } ++ .{false} ** 2;
```

Comment:
I cannot really understand this behavior, why `zig fmt` always adds spaces between curly bracket and their content and here it removed it. Is this part of spec?

Reply:
I believe anonymous struct literals with multiple items are supposed to have spaces on the sides, whereas ones with one item do not.